# Reduction of files
Downloaded the individual approach plates from FAA [website](https://www.faa.gov/air_traffic/flight_info/aeronav/digital_products/dtpp/).  The individual approach plates were used rather than the area pdf's as they were easier to parse.  This notebook removes all plates that are not RNAV approaches.

In [7]:
import os

approach_plate_dir = "./approach_plates"
original_num_files = len(os.listdir(approach_plate_dir))
for filename in os.listdir(approach_plate_dir):
    if filename[5] != "R":
        os.remove(approach_plate_dir + "/" + filename)
final_num_files = len(os.listdir(approach_plate_dir))
print("Of", original_num_files, "files,", original_num_files - final_num_files, "were removed, leaving", final_num_files)

Of 16733 files, 9355 were removed, leaving 7378


## Results
Utilized Jul 13 2023 - Aug 10 2023 Data
16733 files included arrivals, departures, approaches, airport diagrams, and others.
9355 files were removed leaving 7378 RNAV approaches